src/list.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-unused-vars */
+
+const { defaultOptions } = require("./utils");
+const { SkynetClient } = require("./client");
+
+const defaultListFilesOptions = {
+  ...defaultOptions(""),
+  endpointPathListFilesDir: "/renter/dir",
+  endpointPathListFilesFile: "/renter/file",
+};
+
+SkynetClient.prototype.listFiles = function (customOptions = {}) {
```

Comment:
ok wait, I thought it was supposed to be skynet sdk and now it starts to look like our https://gitlab.com/NebulousLabs/sia-typescript

we don't even expose this method on skynet portal nor do I think we have any plans for it...

Reply:
Since we have `siac skynet ls` I thought it would be appropriate to include in the API. We can ask David whether it belongs here.